.github/ISSUE_TEMPLATE/Outbound_Review.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Outbound Review
+about: Ticket for requesting an Outbound Review
+title: Request an Outbound Review
+labels: helpdesk
+assignees: ''
+
+---
+# Outbound Review
```

Comment:
It would be nice to add a description on what an outbound review is to provide the user context